src/04-arrays.js

```diff
@@ -79,7 +89,10 @@ function getLastElementOfArray(array) {}
  *  addElementToEndOfArray([ 10 ], 9);
  *  //> [ 10, 9 ]
  */
-function addElementToEndOfArray(array, element) {}
+function addElementToEndOfArray(array, element) {
+  array.push(element)
+  return array
```

Comment:
Why is it we can not simply write ` return array.push(element) ` ? Make sure we know why this is - this is!